templates/new-design/parts/header.php

```diff
@@ -1,4 +1,52 @@
 <?php
+namespace Wporg\TranslationEvents\Templates\NewDesign\Parts;
```

Comment:
I noticed that `glotpress/assets/css/style.css` is being included on the page, I'm not sure exactly what is causing that. The idea with the `new-design/` directory was that that would be a "clean slate" that doesn't have any of the glotpress styles.

Any idea what's causing this style to be included? It's visible in the screenshot that the style is included because the Translation Events header has the old styling.

Reply:
I think this is because at translate.w.org, GlotPress loads on every call, so it loads its own CSS. It happens the same with other plugins like gp-translation-helpers.